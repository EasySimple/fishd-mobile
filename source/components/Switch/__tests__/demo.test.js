import webDemoTest from '../../../tests/demoTest';

webDemoTest('switch');
