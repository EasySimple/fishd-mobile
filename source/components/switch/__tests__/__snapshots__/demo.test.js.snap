// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders switch demo correctly 1`] = `<div />`;
